lib/Doctrine/ORM/Query/Parser.php

```diff
@@ -842,7 +847,7 @@ public function QueryLanguage()
 
         $this->lexer->moveNext();
 
-        switch ($this->lexer->lookahead['type']) {
+        switch ($this->lexer->lookahead === null ? null : $this->lexer->lookahead['type']) {
```

Comment:
Can also be simplified with the `??` operator

Reply:
@lcobucci `$a === null ? null : $b` is not the same as `$a ?? $b`...

Reply:
I meant `$this->lexer->lookahead['type'] ?? null` https://3v4l.org/Jjead :)

Reply:
Done.